pubtools/pulplib/_impl/model/repository/yum.py

```diff
@@ -70,6 +70,18 @@ class YumRepository(Repository):
 
     type = pulp_attrib(default="rpm-repo", type=str, pulp_field="notes._repo-type")
 
+    population_sources = pulp_attrib(
+        default=(), type=tuple, converter=tuple, pulp_field="notes.population_sources"
+    )
+    """List of repositories used for populate the repository
```

Comment:
```suggestion
    """List of repository IDs used to populate this repository
```

It is repo IDs, right?  Not actually Repository objects.